lms/static/sass/elements/_progress-circle.scss

```diff
@@ -0,0 +1,59 @@
+.program-progress {
+    width: 300px;
+    margin: 0 auto 30px;
+
+    @media(min-width: $bp-screen-md) {
+        margin-left: 0;
+    }
+}
+
+.progress-heading {
+    color: $blue-d1;
```

Comment:
@andy-armstrong would the variables be defined at the top of this partial, or elsewhere?